frontend/src/Utils/FeatureFlagLoader.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { usePluginStore } from '@openshift/dynamic-plugin-sdk';
+import type { SetFeatureFlag } from '@openshift/dynamic-plugin-sdk';
+
+export type ModelFeatureFlagLoaderProps = {
+  handler: (callback: SetFeatureFlag) => void;
+};
+/**
+ * Component to wrap handling of feature flag handler.
+ * @param function function that accepts single argmunet of type SetFeatureFlag callback.
+ * @returns null
+ */
+const FeatureFlagLoader = ({ handler }) => {
+  const pluginStore = usePluginStore();
+  const setFlagCallback = React.useCallback(
+    (name, value) => {
+      pluginStore.setFeatureFlags({ [name]: value });
+    },
+    [pluginStore],
+  );
+  handler(setFlagCallback);
```

Comment:
```suggestion
  React.useEffect(() => handler(setFlagCallback), [handler, setFlagCallback]);
```

Reply:
Same issue as the other pr. `useEffect` will be called after render which does have potential to cause a flicker.

Reply:
Isn't this a bit too much? It's effectively the same as before? Whenever we receive `pluginStore` it will mutate `setFlagCallback`, whenever that gets changed we'd call the handler. Also whenever the `handler` updates it gets called as well. We'd be just adding yet another function wrapper and React calculation on top of it.